packages/sdk/src/core/contextInternal.ts

```diff
@@ -114,23 +135,24 @@ export class ContextInternal {
 
 		// check permission for exclusive actors
 		let user: User;
-		if (actor.exclusiveToUser &&
-			(user = this.userSet.get(actor.exclusiveToUser)) &&
-			!(user.grantedPermissions.includes(Permissions.UserInteraction))) {
-			actor.internal.notifyCreated(false,
-				`Permission denied on user ${user.id} (${user.name}). Either this MRE did not ` +
-				"request the UserInteraction permission, or it was denied by the user."
-			);
+		if (actor.exclusiveToUser) {
+			user = this.userSet.get(actor.exclusiveToUser);
+			//if (user && user.grantedPermissions &&
+			//	!(user.grantedPermissions.includes(Permissions.UserInteraction))) {
+			//	actor.internal.notifyCreated(false,
+			//		`Permission denied on user ${user.id} (${user.name}). Either this MRE did not ` +
+			//		"request the UserInteraction permission, or it was denied by the user.");
+			//}
 		}
 
 		// check permission for attachments
-		if (actor.attachment?.userId &&
-			(user = this.userSet.get(actor.attachment?.userId)) &&
-			!(user.grantedPermissions.includes(Permissions.UserInteraction))) {
-			actor.internal.notifyCreated(false,
-				`Permission denied on user ${user.id} (${user.name}). Either this MRE did not ` +
-				"request the UserInteraction permission, or it was denied by the user."
-			);
+		if (actor.attachment?.userId) {
+			user = this.userSet.get(actor.attachment?.userId);
+			//if (!(user.grantedPermissions.includes(Permissions.UserInteraction))) {
```

Comment:
TODO: this should be fixed before merge

Reply:
Is the code getting uncommented as a part of preparing this for a merge?

---
In reply to: [449561436](https://github.com/microsoft/mixed-reality-extension-sdk/pull/652#discussion_r449561436) [](ancestors = 449561436)

Reply:
Added user.hasGrantedPermissions

---
In reply to: [450333867](https://github.com/microsoft/mixed-reality-extension-sdk/pull/652#discussion_r450333867) [](ancestors = 450333867,449561436)